docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -3405,6 +3461,88 @@ DIFF2_I1TS
 --- 5 row(s) selected.
 ```
 
+<<<
+[[emptyblob_function]]
+== EMPTY_BLOB() Function
+
+[[syntax_descriptions_of_emptyblob_function]]
+=== Syntax Descriptions of EMPTY_BLOB() Function
+
+The EMPTY_BLOB() function creates a dummy LOB handle with BLOB.
+
+This function can be used in INSERT or UPDATE statement, to initialize a LOB column or attribute to EMPTY.
```

Comment:
Either "an INSERT or UPDATE statement" or "INSERT or UPDATE statements" is better

Reply:
"an INSERT or UPDATE statement" is preferable, thanks Dave. :)